CONTRIBUTING.md

```diff
@@ -17,4 +17,43 @@ This project and everyone participating in it is governed by the code of conduct
 ## How can I contribute?
 1. You can report bugs/glitches/faults that you are likely to come across when using the project server and client apps.
 2. You can suggest enhancements that you would want to see implemented on the project - which gets voted in the roadmap.
-3. You can address a wide variety of open issues if you wish to contribute using your code and make a pull request.
\ No newline at end of file
+3. You can address a wide variety of open issues if you wish to contribute using your code and make a pull request.
+
+## Steps to contribute
+
+* Comment on the issue you want to work on. Make sure it's not assigned to someone else.
+
+* If you think you encounter a bug or have a suggestion for improvement of code or to add a feature, then create a issue with proper title and tags but first make sure it's not already present. And if you want to work on that, then comment on that issue and wait for the owner to assign that work to you.
+
+### Making a PR
+
+> - Make sure you have been assigned the issue to which you are making a PR.
+> - If you make PR before being assigned, It will be labeled `invalid` and closed without merging.
```

Comment:
I am so glad that you mentioned this. It has been nothing but a headache for us maintainers to deal with folks who directly jump into making PRs without assignment. This is certainly going to come handy.